<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lab08_PNR</class>
 <widget class="QMainWindow" name="Lab08_PNR">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lab08_PNR</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget_PNR">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>650</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab1_PNR">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QPushButton" name="btnPrint_PNR">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>490</y>
        <width>771</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Print</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="plainTextEdit_PNR">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>771</width>
        <height>411</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lblClasses_PNR">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Classes:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab2_PNR">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QPlainTextEdit" name="plainTextEdit2_PNR">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>771</width>
        <height>411</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSort_PNR">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>450</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Sort</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSet_PNR">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>450</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Set</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxElemAt_PNR">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>530</y>
        <width>101</width>
        <height>29</height>
       </rect>
      </property>
      <property name="maximum">
       <number>53</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditPassengerName_PNR">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>530</y>
        <width>431</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lblPassengerName_PNR">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>510</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Passenger Name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblElemAt_PNR">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>510</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Element at:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditPrice_PNR">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>530</y>
        <width>121</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lblPrice_PNR">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>510</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Price:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblPlace_PNR">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>510</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Place:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxPlace_PNR">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>530</y>
        <width>91</width>
        <height>28</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Bottom</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Upper</string>
       </property>
      </item>
     </widget>
     <widget class="QSpinBox" name="spinBoxCar_PNR">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>580</y>
        <width>81</width>
        <height>29</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>999</number>
      </property>
     </widget>
     <widget class="QLabel" name="lblCar_PNR">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>560</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Car:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxSeat_PNR">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>580</y>
        <width>81</width>
        <height>29</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>999</number>
      </property>
     </widget>
     <widget class="QLabel" name="lblSeat_PNR">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>560</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Seat:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblMsg_PNR">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>580</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Message:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblMsgText_PNR">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>580</y>
        <width>331</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPrint2_PNR">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>450</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Print</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSet_PNR_2">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>450</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Calc Sum</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblSum_PNR">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>560</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Sum:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblSumVal_PNR">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>580</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
